rpyscope/gui_qt.py

```diff
@@ -289,7 +289,7 @@ def __init__(self, parent=None, top=0, cam=None):
         :param top: Top adjuster, where to start, in px, defaults to 0
         :type top: int
         """
-        super(CommandLineScope, self).__init__(parent)
+        super().__init__(parent)
```

Comment:
Again, I didn't read command window code so I just trust you.

Reply:
same as above, this comes from the `py36+` argument in the pre-commit file